test/services/CategorisationServiceSpec.scala

```diff
@@ -307,6 +313,110 @@ class CategorisationServiceSpec extends SpecBase with BeforeAndAfterEach {
 
       }
 
+      "if NIPHL is not authorised and has NIPHL assessments" in {
+        val assessment1 = CategoryAssessment(
+          "ass1",
+          1,
+          Seq(
+            Certificate("WFE012", "cert1code", "cert1desc")
+          )
+        )
+
+        val assessment2 = CategoryAssessment(
+          "ass2",
+          1,
+          Seq(
+            Certificate("WFE012", "cert2code", "cert2desc")
+          )
+        )
+
+        val categorisationInfo = CategorisationInfo(
+          "1234567890",
+          Seq(
+            assessment1,
+            assessment2
+          ),
+          Seq(assessment1, assessment2),
```

Comment:
This should be empty as both items are Niphls and so do not need answering

Reply:
Fixed